freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -626,6 +626,37 @@ def _create_original_to_rebuilt_nvrs_map(self):
                         and block_build_nvr["version"] == parsed_build_nvr["version"]  # noqa: W503
                     ):
                         nvrs_mapping[block_build] = build_nvr
+
+        if self.event.get("autocomplete_field", False): #TODO implement "autocomplete" field of event
+            self._add_parent_nvrs(nvrs_mapping)
```

Comment:
`_add_parent_nvrs` returns a dict, but you didn't use it, the `nvrs_mapping` is actually updated implicitly which is not good.

Reply:
Great point! I'll return a dictionary with the updated items and store it explicitly in `nvrs_mapping`, in order to avoid the implicit udpate.